src/SVG.gd

```diff
@@ -16,7 +16,19 @@ func _ready() -> void:
 	SVG.root_tag.child_attribute_changed.connect(update_text)
 	SVG.root_tag.tag_layout_changed.connect(update_text)
 	
-	if GlobalSettings.save_data.svg_text.is_empty():
+	var opening_file := false
```

Comment:
This variable seems unnecessary.

Also lower down, remove the semicolon.

Reply:
the opening_file variable? It was the best way I could think of lol, how would you have done that?

Reply:
Ah sorry, I didn't see that this was important for the condition below! In that case, just fix the semicolon.

Reply:
you wouldnt happen to know a direct clone from the repo with no changes gives me errors would you lmao?

Reply:
anyway Ill start a new PR because the fork this one is ffrom imploded in on itself